cabal.project

```diff
@@ -42,15 +42,63 @@ packages:
 -- See https://github.com/haskell/haskell-language-server/blob/master/.gitlab-ci.yml
 optional-packages: vendored/*/*.cabal
 
-tests: true
+tests: True
+
+-- mfsolve has duplicate instances in its test suite
+-- See: https://github.com/kuribas/mfsolve/issues/8
+package mfsolve
+  tests: False
+
+if impl(ghc >= 9.5)
+  source-repository-package
+    type:git
+    location: https://github.com/wz1000/HieDb
+    tag: 721bd0ce7392b8eff8d7643cf132a6448466fd52
+
+  source-repository-package
+    type:git
+    location: https://github.com/wz1000/retrie
+    tag: 3ad8ca0450ea4619bbc8007251582ffc828faa37
+
+  source-repository-package
+    type:git
+    location: https://github.com/haskell/hie-bios
+    tag: 8519812ad7501cab31347cd46ad1312b8413b8ad
+
+  source-repository-package
+    type:git
+    location: https://github.com/wz1000/ghc-exactprint/
+    tag: 58d88820399b66304f3bb18f0b2602c3a90d5bea
+
+-- Remove when fix for
+-- https://github.com/kazu-yamamoto/http2/issues/42
+-- is released on hackage
+source-repository-package
+  type:git
+  location: https://github.com/kazu-yamamoto/http2
+  tag: aa56ded3494dd4f0efb0bbcb5378879ce785a647
 
 package *
   ghc-options: -haddock
   test-show-details: direct
 
 write-ghc-environment-files: never
 
-index-state: 2023-01-27T00:00:00Z
+if impl(ghc >= 9.4)
```

Comment:
Should be 9.5?

Reply:
Yes, thanks. I was having issues with some packages on 9.4 that I was considering using head.hackage patches for. Reverted this.